<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.1.189">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>BS3351: Plant Molecular Phylogeny and Taxonomy - 1&nbsp; Molecular Systemics</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<link href="../lessons/lesson2.html" rel="next">
<link href="../index.html" rel="prev">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>
<script async="" src="https://hypothes.is/embed.js"></script>


</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
    <div class="container-fluid d-flex justify-content-between">
      <h1 class="quarto-secondary-nav-title"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Molecular Systemics</span></h1>
      <button type="button" class="quarto-btn-toggle btn" aria-label="Show secondary navigation">
        <i class="bi bi-chevron-right"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="../">BS3351: Plant Molecular Phylogeny and Taxonomy</a> 
        <div class="sidebar-tools-main tools-wide">
    <a href="https://github.com/kevfo/BS3037_Notes/" title="Source Code" class="sidebar-tool px-1"><i class="bi bi-github"></i></a>
    <a href="" title="Share" id="sidebar-tool-dropdown-0" class="sidebar-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false"><i class="bi bi-share"></i></a>
    <ul class="dropdown-menu" aria-labelledby="sidebar-tool-dropdown-0">
        <li>
          <a class="dropdown-item sidebar-tools-main-item" href="https://twitter.com/intent/tweet?url=|url|">
            <i class="bi bi-bi-twitter pe-1"></i>
          Twitter
          </a>
        </li>
        <li>
          <a class="dropdown-item sidebar-tools-main-item" href="https://www.facebook.com/sharer/sharer.php?u=|url|">
            <i class="bi bi-bi-facebook pe-1"></i>
          Facebook
          </a>
        </li>
    </ul>
  <a href="" class="quarto-color-scheme-toggle sidebar-tool" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
  <a href="" class="quarto-reader-toggle sidebar-tool" onclick="window.quartoToggleReader(); return false;" title="Toggle reader mode">
  <div class="quarto-reader-toggle-btn">
  <i class="bi"></i>
  </div>
</a>
</div>
    </div>
      </div>
      <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
      </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../index.html" class="sidebar-item-text sidebar-link">Preface</a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../lessons/lesson1.html" class="sidebar-item-text sidebar-link active"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Molecular Systemics</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../lessons/lesson2.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Plant Taxonomy and Molecular Systematics</span></a>
  </div>
</li>
    </ul>
    </div>
</nav>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#molecular-phylogenetics" id="toc-molecular-phylogenetics" class="nav-link active" data-scroll-target="#molecular-phylogenetics"><span class="toc-section-number">1.1</span>  Molecular Phylogenetics</a>
  <ul class="collapse">
  <li><a href="#homology-and-homoplasy" id="toc-homology-and-homoplasy" class="nav-link" data-scroll-target="#homology-and-homoplasy"><span class="toc-section-number">1.1.1</span>  Homology and Homoplasy</a></li>
  <li><a href="#molecular-data" id="toc-molecular-data" class="nav-link" data-scroll-target="#molecular-data"><span class="toc-section-number">1.1.2</span>  Molecular Data</a></li>
  </ul></li>
  <li><a href="#phylogenetic-trees" id="toc-phylogenetic-trees" class="nav-link" data-scroll-target="#phylogenetic-trees"><span class="toc-section-number">1.2</span>  Phylogenetic Trees</a>
  <ul class="collapse">
  <li><a href="#groupings" id="toc-groupings" class="nav-link" data-scroll-target="#groupings"><span class="toc-section-number">1.2.1</span>  Groupings</a></li>
  <li><a href="#traits" id="toc-traits" class="nav-link" data-scroll-target="#traits"><span class="toc-section-number">1.2.2</span>  Traits</a></li>
  </ul></li>
  </ul>
<div class="toc-actions"><div><i class="bi bi-github"></i></div><div class="action-links"><p><a href="https://github.com/kevfo/BS3037_Notes/edit/main/lessons/lesson1.qmd" class="toc-action">Edit this page</a></p><p><a href="https://github.com/kevfo/BS3037_Notes/issues/new" class="toc-action">Report an issue</a></p></div></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title d-none d-lg-block"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Molecular Systemics</span></h1>
</div>



<div class="quarto-title-meta">

    
    
  </div>
  

</header>

<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/lesson1/relevance.png" class="img-fluid figure-img" width="215"></p>
<p></p><figcaption class="figure-caption">Relevance of Phylogenetics</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Molecular Phylogenetics is all about understanding how different living things are related to each other and how they’ve evolved over time. It helps scientists figure out family trees for all kinds of organisms, giving us the best picture of their evolutionary history. This information helps answer important questions in biology, like how fast species change over time, how they interact with each other, and where they originally came from. Plus, it’s super useful for things like identifying species, sorting them into groups, tracking down pathogens that cause diseases, solving crimes, and using computers to analyze biological data.</p>
<section id="molecular-phylogenetics" class="level2" data-number="1.1">
<h2 data-number="1.1" class="anchored" data-anchor-id="molecular-phylogenetics"><span class="header-section-number">1.1</span> Molecular Phylogenetics</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/lesson1/molecular_phylogenetics.png" class="img-fluid figure-img" width="212"></p>
<p></p><figcaption class="figure-caption">Illustration of Molecular Phylogenetics by EMBI</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Molecular Phylogenetics</strong> delves into the intricate tapestry of evolutionary history and ancestral connections among organisms, leveraging molecular data for insights.</p>
<p>Within this field, phylogenetics focuses on identifying <strong>homology</strong>, which refers to similarities resulting from inheritance from a shared common ancestor, serving as evidence of shared ancestry among organisms. Researchers construct phylogenetic trees through meticulous analysis of molecular sequences, enabling them to infer relationships among various entities, be it species, individuals within a population, or even genes. These trees visually represent the evolutionary relationships, offering a snapshot of the complex web of life’s interconnectedness and evolutionary trajectories.</p>
<section id="homology-and-homoplasy" class="level3" data-number="1.1.1">
<h3 data-number="1.1.1" class="anchored" data-anchor-id="homology-and-homoplasy"><span class="header-section-number">1.1.1</span> Homology and Homoplasy</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/lesson1/horses.png" class="img-fluid figure-img" width="242"></p>
<p></p><figcaption class="figure-caption">Example of Homology Using Horses</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Homology</strong> refers to similarities among organisms that stem from shared ancestry, providing evidence of common lineage. This similarity is fundamental in reconstructing evolutionary history, with fossil data playing a key role.</p>
<p>However, fossilization is an exceedingly rare process, leading to a scarcity of known fossils for many taxa. Additionally, even when fossils are available, they may not definitively align with specific lineages within taxa, introducing ambiguity into the evolutionary narrative. This complexity underscores the challenges researchers face in unraveling the intricate tapestry of life’s evolutionary pathways.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/lesson1/phylo2.png" class="img-fluid figure-img" width="238"></p>
<p></p><figcaption class="figure-caption">Example Phylogenetic Tree Constructed Using Fossil Evidence</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>In reconstructing evolutionary histories using present-day data, scientists employ a method that involves grouping organisms based on shared traits. However, it’s essential to only compare traits that originate from a common ancestor, known as <strong>homologous traits</strong>. This ensures that the comparisons accurately reflect the evolutionary relationships between different organisms. By focusing on these homologous traits, researchers can effectively piece together the complex puzzle of how species are related and have evolved over time.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/lesson1/homoplasy.png" class="img-fluid figure-img" width="254"></p>
<p></p><figcaption class="figure-caption">Example of Homoplasy</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>When a trait evolves independently in multiple groups, it’s termed <strong>homoplasy</strong>. Thus, distinguishing between homologous traits and those resulting from convergent evolution is critical in accurately understanding evolutionary relationships.</p>
</section>
<section id="molecular-data" class="level3" data-number="1.1.2">
<h3 data-number="1.1.2" class="anchored" data-anchor-id="molecular-data"><span class="header-section-number">1.1.2</span> Molecular Data</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/lesson1/data.png" class="img-fluid figure-img" width="462"></p>
<p></p><figcaption class="figure-caption">Example of Molecular Data</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>A <strong>homologous character</strong> refers to a column in the data alignment, whether it’s DNA, RNA, or amino acids. This column represents a specific position in the sequence alignment where the molecules share a common ancestry.</p>
<section id="why-use-molecular-data" class="level4" data-number="1.1.2.1">
<h4 data-number="1.1.2.1" class="anchored" data-anchor-id="why-use-molecular-data"><span class="header-section-number">1.1.2.1</span> Why Use Molecular Data?</h4>
<p>DNA and amino acid sequences serve as inherited units, carrying genetic information across generations. DNA data, in particular, is remarkably specific and information-rich.</p>
<p>Thanks to modern techniques, sequencing DNA has become quick, reliable, and inexpensive. This allows for the unambiguous description of characters and quantitative traits encoded within the genetic material.</p>
<p>Furthermore, there’s a growing body of knowledge surrounding the molecular evolution of DNA, enriching our understanding of how genetic information changes over time.</p>
</section>
</section>
</section>
<section id="phylogenetic-trees" class="level2" data-number="1.2">
<h2 data-number="1.2" class="anchored" data-anchor-id="phylogenetic-trees"><span class="header-section-number">1.2</span> Phylogenetic Trees</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/lesson1/tree1.png" class="img-fluid figure-img" width="490"></p>
<p></p><figcaption class="figure-caption">Diagram of a Phylogenetic Tree</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>We often use a visual representation called a <strong>phylogenetic tree</strong> to show evolutionary relationships. The tree consists of nodes connected by branches. A <strong>branch</strong> represents an evolutionary lineage, while a <strong>node</strong> represents a hypothetical common ancestor. <strong>Nodes</strong> occur where a lineage splits into two, such as during a speciation event when one species diverges into two separate species.</p>
<p>The <strong>root</strong> of the tree is an internal node representing the most recent common ancestor of all the species being studied. Think of it as the starting point of the tree. An <strong>outgroup</strong> is a taxon chosen as an external point of reference, usually a species or group of species that diverged from the rest of the group being studied earlier in evolutionary history. A <strong>clade</strong> is a group of organisms that have all descended from a common ancestor.</p>
<p>Understanding the root is essential because it helps us infer the order of nodes through time. Without knowing the root, it’s challenging to determine the sequence of evolutionary events accurately.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/lesson1/phylo3.png" class="img-fluid figure-img" width="284"></p>
<p></p><figcaption class="figure-caption">Distance and Probabilities on a Phylogenetic Tree</figcaption><p></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/lesson1/phylo4.png" class="img-fluid figure-img" width="280"></p>
<p></p><figcaption class="figure-caption">Distance and Probabilities on a Phylogenetic Tree</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>In molecular phylogenetics, the values associated with branches and nodes provide important information about the evolutionary relationships depicted in a phylogenetic tree. Branch lengths can vary in their informativeness. In a cladogram, where branch lengths are uninformative, the tree represents only the branching pattern of relationships, without indicating the amount of genetic change that occurred along each branch. However, in a phylogram, branch lengths provide informative data about the amount of genetic change between nodes.</p>
<p>To interpret branch lengths, researchers often use a scale bar. This bar represents the amount of genetic change, typically measured as substitutions per site. The length of the scale bar corresponds to the number of changes divided by the length of the sequence. Alternatively, it can represent a percentage change. This information helps scientists understand the genetic divergence between different species or groups depicted in the phylogenetic tree.</p>
<p><strong>Bootstrap support</strong> is one measure of nodal support, typically ranging from 0 to 100. A bootstrap support value between 70-80 suggests weak support for the node, 80-90 indicates moderate support, and 90-100 signifies strong support.</p>
<p>Another measure is <strong>Bayesian posterior probabilities</strong>, which also range from 0 to 1. A posterior probability of 0.9-0.95 indicates weak support, while a value of 0.95-1 suggests strong support for the node.</p>
<section id="groupings" class="level3" data-number="1.2.1">
<h3 data-number="1.2.1" class="anchored" data-anchor-id="groupings"><span class="header-section-number">1.2.1</span> Groupings</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/lesson1/mono.png" class="img-fluid figure-img" width="279"></p>
<p></p><figcaption class="figure-caption">Monophyletic Groupings on a Phylogenetic Tree</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The term “<strong>monophyletic</strong>” is synonymous with “clade” and refers to a group of organisms that includes all the descendants of a common ancestor.</p>
<p>A group that excludes some of its descendants is termed “<strong>paraphyletic</strong>.” An example of this is reptiles excluding birds.</p>
<p>If a group is <strong>polyphyletic</strong>, this means that it also includes distantly-related phyla.</p>
</section>
<section id="traits" class="level3" data-number="1.2.2">
<h3 data-number="1.2.2" class="anchored" data-anchor-id="traits"><span class="header-section-number">1.2.2</span> Traits</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/lesson1/traits.png" class="img-fluid figure-img" width="245"></p>
<p></p><figcaption class="figure-caption">Traits of Phylogenetic Traits</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Here are some key terms related to traits:</p>
<ul>
<li><strong>Apomorphy</strong>: A derived trait, meaning a characteristic that has evolved in a particular lineage and is different from the ancestral form.</li>
<li><strong>Synapomorphy</strong>: A derived trait shared by members of a clade, indicating common ancestry. For example, seeds in seed plants are a synapomorphy.</li>
<li><strong>Autapomorphy</strong>: A derived trait unique to a single taxon, distinguishing it from other related taxa.</li>
<li><strong>Plesiomorphy</strong>: An ancestral trait, meaning a characteristic that was present in the common ancestor of a group of organisms. For instance, exothermy in “reptiles” is a plesiomorphy.</li>
<li><strong>Symplesiomorphy</strong>: A shared ancestral trait, present in both the ancestor and several descendants, but not unique to any particular group.</li>
<li><strong>Homoplasy</strong>: A trait that evolved independently in multiple groups, leading to similarities between taxa that are not due to common ancestry. For example, fleshy fruits evolving independently in different plant lineages are considered homoplastic traits.</li>
</ul>


</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  let localAlternateSentinel = 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } 
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../index.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text">Preface</span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../lessons/lesson2.html" class="pagination-link">
        <span class="nav-page-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Plant Taxonomy and Molecular Systematics</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->



</body></html>